weblab_cg/templates/chaste/normal_model.cpp

```diff
@@ -0,0 +1,148 @@
+//! @file
+//! 
+//! This source file was generated from CellML.
+//! 
+//! Model: {{model_name_from_file}}
+//! 
+//! Processed by webalab_cg - CellML Tools in Python
+//!     (translators: , webalab_cg: , options: normal)
+//! on {{generation_date}}
+//! 
+//! <autogenerated>
+
+#include "{{model_name_from_file}}.hpp"
+#include <cmath>
+#include <cassert>
+#include <memory>
+#include "Exception.hpp"
+#include "OdeSystemInformation.hpp"
+#include "RegularStimulus.hpp"
+#include "HeartConfig.hpp"
+#include "IsNan.hpp"
+#include "MathsCustomFunctions.hpp"
+	{%- if default_stimulus_equations.membrane_stimulus_current_duration is defined %}
+
+    boost::shared_ptr<RegularStimulus> {{class_name}}FromCellML::UseCellMLDefaultStimulus()
+    {
+        // Use the default stimulus specified by CellML metadata
+		{% for key, value in default_stimulus_equations.items() %}
```

Comment:
Mixing tabs and spaces: Maybe check all files for this! (And see if you can get your IDE to warn you when this happens?)